Thu Nov 29 2018 14:08:08 GMT+0530 (India Standard Time) - SyntaxError: Unexpected string in JSON at position 34
    at JSON.parse (<anonymous>)
    at parse (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\raw-body\index.js:224:16)
    at done (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:182:13)
    at endReadableNT (_stream_readable.js:1090:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
Thu Nov 29 2018 14:08:15 GMT+0530 (India Standard Time) - Error: Could not connect to database
    at mongoose.connect.then.catch (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\utilities\connection.js:50:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Thu Nov 29 2018 14:11:51 GMT+0530 (India Standard Time) - Error: Could not connect to database
    at mongoose.connect.then.catch (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\utilities\connection.js:52:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Thu Nov 29 2018 14:28:15 GMT+0530 (India Standard Time) - Error: No transaction details found
    at dbLayer.getTransactions.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:35:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Thu Nov 29 2018 14:29:01 GMT+0530 (India Standard Time) - Error: No transaction details found
    at dbLayer.getTransactions.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:35:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Thu Nov 29 2018 14:29:19 GMT+0530 (India Standard Time) - Error: No transaction details found
    at dbLayer.getTransactions.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:35:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Thu Nov 29 2018 14:31:46 GMT+0530 (India Standard Time) - TypeError: Cannot read property 'transactions' of null
    at collection.findOne.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\model\account.js:44:25)
Thu Nov 29 2018 14:33:09 GMT+0530 (India Standard Time) - Error: No transaction details found
    at dbLayer.getTransactions.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:35:23)
Thu Nov 29 2018 14:34:03 GMT+0530 (India Standard Time) - Error: No transaction details found
    at dbLayer.getTransactions.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:35:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Thu Nov 29 2018 14:35:15 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 14:35:49 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 14:37:59 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 14:40:12 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 14:40:31 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 14:42:02 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 14:59:00 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: username_1 dup key: { : "guru@gmail.com" }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:06:15 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:11:53 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:13:29 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:17:10 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:43:42 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:44:53 GMT+0530 (India Standard Time) - Error: Invalid PAN number
    at Object.validator.validatePAN (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\utilities\validator.js:13:19)
    at Object.service.createAccount (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:43:15)
    at routing.post (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\routes\routing.js:50:13)
    at Layer.handle [as handle_request] (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:335:12)
    at next (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:335:12)
Thu Nov 29 2018 15:45:00 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:47:07 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:49:25 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:49:41 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:50:02 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:50:15 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:53:24 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 15:53:41 GMT+0530 (India Standard Time) - MongoError: E11000 duplicate key error collection: ExpressWalletDB.Account index: transactions.tid_1 dup key: { : null }
    at Function.create (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\error.js:43:12)
    at toError (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\utils.js:149:22)
    at coll.s.topology.insert (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb\lib\operations\collection_ops.js:844:39)
    at C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\node_modules\mongodb-core\lib\connection\pool.js:532:18
    at process._tickCallback (internal/process/next_tick.js:61:11)
Thu Nov 29 2018 16:07:48 GMT+0530 (India Standard Time) - Error: User already exists
    at dbLayer.getUser.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:46:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Fri Nov 30 2018 14:31:15 GMT+0530 (India Standard Time) - ReferenceError: response is not defined
    at collection.updateOne.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\model\account.js:84:13)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Fri Nov 30 2018 14:32:03 GMT+0530 (India Standard Time) - Error: No transaction details found or operation failed
    at dbLayer.deleteTransaction.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:80:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Fri Nov 30 2018 14:33:12 GMT+0530 (India Standard Time) - Error: No transaction details found or operation failed
    at dbLayer.deleteTransaction.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:80:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Fri Nov 30 2018 15:58:22 GMT+0530 (India Standard Time) - Error: Transaction details not updated
    at dbLayer.updateTransactions.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:69:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Fri Nov 30 2018 15:59:28 GMT+0530 (India Standard Time) - Error: Transaction details not updated
    at dbLayer.updateTransactions.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:69:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Fri Nov 30 2018 15:59:58 GMT+0530 (India Standard Time) - Error: Transaction details not updated
    at dbLayer.updateTransactions.then (C:\Users\gurumoorthyg.g\Desktop\ExpressWalletUpdatedDB\src\service\account.js:69:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Wed Jul 22 2020 - Error: Could not connect to database
    at C:\Users\veena.m02\Desktop\assignment2\ExpressWalletUpdatedDB\src\utilities\connection.js:55:21
Wed Jul 22 2020 - Error: Could not connect to database
    at C:\Users\veena.m02\Desktop\assignment2\ExpressWalletUpdatedDB\src\utilities\connection.js:55:21
Wed Jul 22 2020 - Error: Could not connect to database
    at C:\Users\veena.m02\Desktop\assignment2\ExpressWalletUpdatedDB\src\utilities\connection.js:55:21
Wed Jul 22 2020 - Error: Could not connect to database
    at C:\Users\veena.m02\Desktop\assignment2\ExpressWalletUpdatedDB\src\utilities\connection.js:55:21
Wed Jul 22 2020 - Error: Could not connect to database
    at C:\Users\veena.m02\Desktop\assignment2\ExpressWalletUpdatedDB\src\utilities\connection.js:55:21
Wed Jul 22 2020 - Error: Could not connect to database
    at C:\Users\veena.m02\Desktop\assignment2\ExpressWalletUpdatedDB\src\utilities\connection.js:55:21
Wed Jul 22 2020 - Error: Could not connect to database
    at C:\Users\veena.m02\Desktop\assignment2\ExpressWalletUpdatedDB\src\utilities\connection.js:55:21
Wed Jul 22 2020 - Error: Could not connect to database
    at C:\Users\veena.m02\Desktop\assignment2\ExpressWalletUpdatedDB\src\utilities\connection.js:55:21
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - CastError: Cast to Number failed for value "search" at path "id" for model "posts"
    at model.Query.exec (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\query.js:4351:21)
    at model.Query.Query.then (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\query.js:4443:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:107:49
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - CastError: Cast to Number failed for value "search" at path "id" for model "posts"
    at model.Query.exec (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\query.js:4351:21)
    at model.Query.Query.then (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\query.js:4443:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:107:49
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - CastError: Cast to Number failed for value "search" at path "id" for model "posts"
    at model.Query.exec (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\query.js:4351:21)
    at model.Query.Query.then (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\query.js:4443:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:107:49
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - CastError: Cast to Number failed for value "search" at path "id" for model "posts"
    at model.Query.exec (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\query.js:4351:21)
    at model.Query.Query.then (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\query.js:4443:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:107:49
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User does not exist
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Jul 27 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: Incorrect password
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:29:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Tue Jul 28 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: Incorrect password
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:29:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: Incorrect password
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:29:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:42:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: Incorrect password
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:29:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:164:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:164:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:164:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:164:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:164:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:164:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:164:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:164:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:165:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - ReferenceError: data is not defined
    at Object.model.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:165:42)
    at Object.service.NewdataForEmail (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:126:20)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:122:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
Wed Jul 29 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:129:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Wed Jul 29 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:129:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:129:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:129:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:178:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:178:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:179:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:179:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:179:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - ReferenceError: postsobj is not defined
    at Object.service.updatePosts (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:138:42)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:135:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:284:7
Thu Jul 30 2020 - ReferenceError: postsobj is not defined
    at Object.service.updatePosts (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:138:42)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:135:13
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:284:7
Thu Jul 30 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:198:25
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:198:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:142:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:142:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: data.find is not a function
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:204:29
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:142:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:136:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:143:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:137:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:143:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:137:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:143:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:137:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:143:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:137:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:198:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:143:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:137:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:143:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:137:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:143:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:137:21
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error: Failed to add post
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:144:27
    at CoreDocumentArray.map (<anonymous>)
    at CoreDocumentArray.map (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\types\documentarray.js:42:27)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:137:21
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - ReferenceError: log is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:161:5
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:335:12)
Thu Jul 30 2020 - ReferenceError: log is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:161:5
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:335:12)
Thu Jul 30 2020 - ReferenceError: log is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:161:5
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:335:12)
Thu Jul 30 2020 - ReferenceError: log is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:161:5
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:335:12)
Thu Jul 30 2020 - ReferenceError: log is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:161:5
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:281:22
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:354:14)
    at param (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:365:14)
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:410:3)
    at next (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:174:3)
    at router (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:317:13)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\router\index.js:335:12)
Thu Jul 30 2020 - TypeError: Cannot read property 'map' of undefined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:222:31
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'map' of undefined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:222:31
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'map' of undefined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:222:31
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'map' of undefined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:222:31
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'map' of undefined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:223:31
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'map' of undefined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:223:31
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - TypeError: Cannot read property 'map' of undefined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:223:31
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.send (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:267:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.send (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:267:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.send (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:267:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Thu Jul 30 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sat Aug 01 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sat Aug 01 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sat Aug 01 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.send (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:267:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.send (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:267:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:21
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:21
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - ReferenceError: newdata is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:234:17
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:223:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - ReferenceError: arr is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:234:17
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:223:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.send (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:267:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:170:21
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:165:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error: No post found for this Tag
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:168:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.send (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:267:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.send (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:170:12)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:267:15)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:536:11)
    at ServerResponse.header (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:771:10)
    at ServerResponse.json (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\express\lib\response.js:264:10)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:167:13
    at Array.map (<anonymous>)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\routes\routing.js:164:14
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error: Posts details not updated
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:154:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error: Posts details not updated
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:154:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:194:25
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:194:25
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error: Posts details not updated
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:154:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error: Posts details not updated
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:154:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error: Posts details not updated
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:154:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error: Posts details not updated
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:154:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - Error: Posts details not updated
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:154:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:194:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:194:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Sun Aug 02 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:194:25
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:194:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - TypeError: Cannot read property 'Posts' of null
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:194:25
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ValidationError: newposts validation failed: name: Cast to string failed for value "[ { userName: 'ABC' } ]" at path "name"
    at model.Document.invalidate (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:2575:32)
    at model.$set (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:1247:12)
    at model._handleIndex (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:980:14)
    at model.$set (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:921:22)
    at model.Document (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:140:12)
    at model.Model (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:106:12)
    at new model (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:4677:15)
    at Array.<anonymous> (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:3253:17)
    at _start (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\helpers\parallelLimit.js:27:37)
    at parallelLimit (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\helpers\parallelLimit.js:23:5)
    at Function.Model.$__insertMany (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:3283:3)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
Mon Aug 03 2020 - ValidationError: newposts validation failed: name: Cast to string failed for value "[ { userName: 'ABC' } ]" at path "name"
    at model.Document.invalidate (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:2575:32)
    at model.$set (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:1247:12)
    at model._handleIndex (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:980:14)
    at model.$set (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:921:22)
    at model.Document (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:140:12)
    at model.Model (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:106:12)
    at new model (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:4677:15)
    at Array.<anonymous> (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:3253:17)
    at _start (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\helpers\parallelLimit.js:27:37)
    at parallelLimit (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\helpers\parallelLimit.js:23:5)
    at Function.Model.$__insertMany (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:3283:3)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
Mon Aug 03 2020 - ValidationError: newposts validation failed: name: Cast to string failed for value "[ { userName: 'ABC' } ]" at path "name"
    at model.Document.invalidate (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:2575:32)
    at model.$set (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:1247:12)
    at model._handleIndex (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:980:14)
    at model.$set (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:921:22)
    at model.Document (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\document.js:140:12)
    at model.Model (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:106:12)
    at new model (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:4677:15)
    at Array.<anonymous> (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:3253:17)
    at _start (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\helpers\parallelLimit.js:27:37)
    at parallelLimit (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\helpers\parallelLimit.js:23:5)
    at Function.Model.$__insertMany (C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\mongoose\lib\model.js:3283:3)
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\node_modules\kareem\index.js:369:33
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:223:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:223:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:223:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:225:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:225:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:225:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:229:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:229:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - ReferenceError: userName is not defined
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\model\model.js:229:44
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:31:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:31:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:31:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User already exists
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:49:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:31:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:31:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:31:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:31:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: Incorrect password
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:29:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Mon Aug 03 2020 - Error: User does not exist.Please create an account to continue
    at C:\Users\veena.m02\Desktop\ass2\ExpressWalletUpdatedDB\src\service\service.js:24:23
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
